tests/test_load.py

```diff
@@ -26,9 +26,26 @@
 
 
 def clear_root(root: str):
-    root = audeer.path(root)
-    if os.path.exists(root):
-        shutil.rmtree(root)
+    audeer.rmdir(root)
+
+
+@pytest.fixture(
+    scope='function',
+    autouse=True,
+)
+def ensure_tmp_folder_deleted():
+    """Fixture to test that the ~ tmp folder gets deleted.
+
+    audb.load() first loads files to a folder
+    named after the database
+    and appended by ``'~'``.
+    This folder should be deleted in the end.
+
+    """
+    yield
+
+    dirs = audeer.list_dir_names(pytest.CACHE_ROOT, recursive=True)
```

Comment:
This might be slow if we have a large database in cache with lots of sub-folders.
Do you think we can simply assume this will never be the case in our unit tests, or should we look for another way to search for potential tmp folders?

Reply:
No, I don't think we will ever unit test a database large enough that this will be a bottleneck.

Reply:
I would propose to stick with this solution and merge. Do you agree?

Reply:
Yes, please go ahead.